app/src/Controller/Api/GameController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Controller\Api;
+
+use App\Entity\Game;
+use App\GameManipulator;
+use App\Repository\GameRepository;
+use App\Repository\ShipRepository;
+use App\Twig\AppExtension;
```

Comment:
Do you need Twig?

Reply:
Yes, for the ships grid attributes. It helps me place them on the board in the frontend.